pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -66,6 +66,94 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         return {parity_high, parity_middle, parity_low};
     }
 
+    static std::tuple<size_t, size_t, size_t, size_t>
```

Comment:
How about changing the return type to `std::array<size_t, 4>` (this one as well as other `revWireParity` functions)?

Reply:
Good idea! I'm about to implement it in this way and `std::sort` was adopted.